releasenotes/notes/use-new-serialization-for-events-and-servicechecks-2b9d2f1331444637.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fix 'Error code "413 Request Entity Too Large" received while sending transaction to "https://dd.agent.datadoghq.eu/api/v1/check_run?api_key=*************************28e95": Payload too big, dropping it'
```

Comment:
```suggestion
    Fixed a few edge cases that could lead to service checks payloads being rejected by Datadog's intake for being too big
```

Reply:
I cherry-pick the wrong commit. Good catch.